packages/react-sdk/src/utils.tsx

```diff
@@ -0,0 +1,61 @@
+import hoistNonReactStatics = require('hoist-non-react-statics')
```

Comment:
Renamed this file to `.tsx` extension because it uses JSX now, and added `hoistStaticsAndForwardRefs` method